docs/using-workflows/adding-connections.md

```diff
@@ -22,3 +24,24 @@ To set the required values for the connection, on the workflow's page, expand th
 ## Managing connections 
 Connections can be managed from the Connections menu option on the left nav. You can add new connections, override the values for existing connections, or delete existing connections. 
 
+## About Connections
+Connections are encrypted when entered and remain encrypted until they are used by a workflow.
+
+When workflows are run, workflow actions that require a connection request the connection value from the Relay Interace. Secrets can be accessed from within an action using one of the Relay SDKs:
+
+```
+relay = Interface()
+sess = boto3.Session(
+  aws_access_key_id=relay.get(D.aws.connection.accessKeyID),
+  aws_secret_access_key=relay.get(D.aws.connection.secretAccessKey),
+  region_name=relay.get(D.aws.region),
+)
+```
+[Example AWS action](https://github.com/relay-integrations/aws-ec2/blob/master/actions/steps/ec2-describe-instances/step.py)
+
+Connections are global to an account and can be referenced by other workflows. 
+
+As any action or workflow can request a connection, some best practices should be followed: 
+- When creating credentials, grant the minimum possible credentials for the workflow to run. Workflow connection requirements should be documented. 
```

Comment:
What does the second sentence really mean? What kind of requirements are there, and where are you supposed to document them?